src/pages/instances/InstanceList.tsx

```diff
@@ -17,7 +18,7 @@ import { instanceCreationTypes } from "util/instanceOptions";
 import InstanceStatusIcon from "./InstanceStatusIcon";
 import TableColumnsSelect from "components/TableColumnsSelect";
 import useEventListener from "@use-it/event-listener";
-import EmptyState from "components/EmptyState";
+
```

Comment:
Why leave an empty line here, I think this should be a single block - also in the other files.